File: app/src/main/java/xyz/junerver/testcaptureandrecord/LogUtils.java
package xyz.junerver.testcaptureandrecord;

import android.util.Log;

/**
 * Description:
 *
 * @author Junerver
 * date: 2022/6/20-9:37
 * Email: junerver@gmail.com
 * Version: v1.0
 */
class LogUtils {
    private static final String TAG = "CaptureScreen";
    public static void d( String msg) {
        Log.d(TAG, msg);
    }

    public static void e( String msg) {
        Log.e(TAG, msg);
    }
}

File: app/src/main/java/xyz/junerver/testcaptureandrecord/LogUtils.kt
package xyz.junerver.testcaptureandrecord

import android.util.Log
import xyz.junerver.testcaptureandrecord.LogUtils

/**
 * Description:
 *
 * @author Junerver
 * date: 2022/6/20-9:37
 * Email: junerver@gmail.com
 * Version: v1.0
 */
object LogUtils {
    private const val TAG = "CaptureScreen"
    @JvmStatic
    fun d(msg: String?) {
        Log.d(TAG, msg!!)
    }

    @JvmStatic
    fun e(msg: String?) {
        Log.e(TAG, msg!!)
    }
}